addons/l10n_be_pos_sale/static/src/js/PaymentScreen.js

```diff
@@ -16,11 +16,15 @@ patch(PaymentScreen.prototype, {
         });
     },
     toggleIsToInvoice() {
-        if (this.checkIsToInvoice()) {
+        const orderLines = this.currentOrder.get_orderlines();
+        const has_intracom_taxes = orderLines.some((line) =>
+            line.tax_ids?.some((tax) => this.pos.intracom_tax_ids.includes(tax.id))
+        );
```

Comment:
I'm assuming this extra condition shouldn't affect checkIsToInvoice which determines if, by default, an order should be invoiced. Keeping it out of that method is coherent for now, as whenever an invoice cannot be turned off it will be on by default.